eng/build.ps1

```diff
@@ -51,12 +49,6 @@ if ($bundletools) {
     $test = $False
 }
 
-# Remove the private build registry keys
```

Comment:
Out of curiosity, why is `PrivateBuildPath` being removed? Should we also remove it from [`eng/build.sh`](https://github.com/dotnet/diagnostics/blob/main/eng/build.sh#L72-L74)

Reply:
It was used to testing a privately built version of the runtime. It was really never used and it was really broken.  It looks like I forgot to remove the support from build.sh.  I do that.